### PR TITLE
KEYCLOAK-12173 removed escaping of '&'

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

**Review by keaz — DISMISSED** [review]

**Review by ssilvert — CHANGES_REQUESTED** [review]
@vmuzikar States that both JIRAs should be fixed before making this test change.  So do we want to wait and make this change after KEYCLOAK-12189 is done?

File: testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/ReferrerTest.java [hunk 1]
```diff
@@ -163,8 +163,6 @@ private String getFakeClientUrl() {
         // we need to use some page which host exists – Firefox is throwing exceptions like crazy if we try to load
         // a page on a non-existing host, like e.g. http://non-existing-server/
         // also we need to do this here as getAuthServerRoot is not ready when firing this class' constructor
-        return getAuthServerRoot() + "auth/non-existing-page/?foo=bar";
-        // TODO replace ^^ with the following once KEYCLOAK-12173 and KEYCLOAK-12189 are resolved
-        // return getAuthServerRoot() + "auth/non-existing-page/?foo=bar&bar=foo#anchor";
+         return getAuthServerRoot() + "auth/non-existing-page/?foo=bar&bar=foo#anchor";
```
Comment (ssilvert) [inline testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/ReferrerTest.java:166]: Referring to this change ^^^

Comment (edewit) [reply testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/ReferrerTest.java:166]: when test passes doesn't that mean that this fix fixes both issues?

**Review by vmuzikar — COMMENTED** [review]
@edewit This PR seems to be affected by https://issues.redhat.com/browse/KEYCLOAK-13379. Could you please rebase so we could test it?

**Comment by edewit** [comment]
@vmuzikar sure, done

File: testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/ReferrerTest.java (hunk 1 above)
Comment (vmuzikar) [reply testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/ReferrerTest.java:166]: We currently no way how to test [KEYCLOAK-12189](https://issues.redhat.com/browse/KEYCLOAK-12189) as i18n support was temporarily removed from the new Account Console and the test related to [KEYCLOAK-12189](https://issues.redhat.com/browse/KEYCLOAK-12189) was [ignored](https://github.com/keycloak/keycloak/blob/e731736d7e37d4e49279746d736c223c15da541c/testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/ReferrerTest.java#L130) for now. So even though we cannot confirm [KEYCLOAK-12189](https://issues.redhat.com/browse/KEYCLOAK-12189) was fixed by this PR, it's safe to do this change.

**Review by vmuzikar — APPROVED** [review]
@edewit Looks good, tests passed: https://keycloak-jenkins.com/job/keycloak-account2-matrix/843/. Thanks!

**Review by ssilvert — APPROVED** [review]

**Pull request merged**